src/app/listArea/listArea.component.html

```diff
@@ -17,7 +17,7 @@
 
   <div class="checkListItem">
     <div>
-      <input type="checkbox">
+      <input class="checkbox" type="checkbox">
```

Comment:
class and type should not be the same name. Update classname to something like `checkbox-input`